microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -1,48 +1,175 @@
+
+For a new WebView2 app, you don't have to know about the old way.
+The key difference between the old and new ways: think about data cleanup during uninstall.<!--doc that.-->
+
+<!-- as maybe stated elsewhere in this article:
+Scenarios for using user data folder, in general; not changing:
+* Uninstalling the app and you want to clean up the data.
+* Deleting the user data folder; resetting the WebView2. -->
+
+Cases where the WebView2 runtime doesn't change the default location:
+
+*  If your WebView2 app already specifies the location of the user data folder, the WebView2 Runtime doesn't change the default location.
+<!-- better?
+*  If your WebView2 app already specifies the location of the user data folder, the WebView2 Runtime doesn't change the location to use the default location instead.
+-->
+
+*  If you don't update your WebView2 SDK, the WebView2 Runtime doesn't change the default location.
+<!-- better?
+If you don't update your WebView2 SDK, the WebView2 Runtime doesn't change the location to use the default location instead.
+-->
+
+*  If you run your WebView2 app before the change<!-- clarify --> and WebView2 created the user data folder for you, the WebView2 Runtime uses that user data folder location;
+   the WebView2 Runtime doesn't change the default user data folder location.
+   <!-- better?:
+   the WebView2 Runtime doesn't change the location of the user data folder to use the default location instead.
+   -->
+
+
+### APIs to retrieve the user data folder location
+
+To retrieve the user data folder location for an existing WebView2 app, use either of the following API items:
+
+* Existing method: [CoreWebView2Environment.CreateAsync method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createasync) - You can use the `userDataFolder` parameter.
+<!-- verbose link, probably don't need:
+/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createasync?view=webview2-dotnet-1.0.1072.54#microsoft-web-webview2-core-corewebview2environment-createasync(system-string-system-string-microsoft-web-webview2-core-corewebview2environmentoptions) -->
+* New property: [CoreWebView2Environment.UserDataFolder property](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder) - You can use this read-only property to see what the user data folder was set to.  This property returns the existing user data folder location.
+<!-- verbose link, probably don't need:
+/dotnet/api/microsoft.web.webview2.core.corewebview2environment.userdatafolder?view=webview2-dotnet-1.0.1072.54
+-->
+
+
+<!-- ====================================================================== -->
+## Specifying another location
+
+tbd
```

Comment:
That is what the create async call above is doing, that is setting where the developer wants to put the UDF